gettime.c

```diff
@@ -693,6 +698,12 @@ int fio_monotonic_clocktest(int debug)
 	if (debug)
 		log_info("cs: Testing %u CPUs\n", nr_cpus);
 
+	/*
+	 * Since multiple threads modify the variable seq on purpose, avoid
+	 * that thread checking tools complain about these accesses.
+	 */
+	DRD_IGNORE_VAR(seq);
```

Comment:
It is probably right to warn on this one. Here's what one of the author's of ThreadSanitizer said: https://groups.google.com/d/msg/thread-sanitizer/XzNS5pRTaE8/ivMDq7ybAgAJ

> > Doesn't the fact that the write happens atomically mean the fact the read 
> > might be non-atomic is safe? 
>
> Hi, 
>
> C standard is pretty clear on this -- no, it's not safe, both accesses 
> must be atomic.